imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -392,4 +388,817 @@ lo_nsw_species-heplus:
 lo_nsw_species-cnoplus:
     <<: *counters
     CATDESC: CNO+ Non-sunward Species
-    FIELDNAM: NSW - CNO+
\ No newline at end of file
+    FIELDNAM: NSW - CNO+
+
+
+# <=== Housekeeping Attributes ===>
+version:
+    <<: *default
+    CATDESC: CCSDS Packet Version Number (always 0)
+    FIELDNAM: Version
+    LABLAXIS: VERSION
+    VAR_TYPE: support_data
+
+type:
+    <<: *default
+    CATDESC: CCSDS Packet Type Indicator (0=telemetry)
+    FIELDNAM: Type
+    LABLAXIS: TYPE
+    VAR_TYPE: support_data
+
+sec_hdr_flg:
+    <<: *default
+    CATDESC: CCSDS Packet Secondary Header Flag (always 1)
+    FIELDNAM: Secondary Header Flag
+    LABLAXIS: SEC_HDR_FLG
+    VAR_TYPE: support_data
+
+pkt_apid:
+    <<: *default
+    CATDESC: CCSDS Packet Application Process ID
+    FIELDNAM: Packet APID
+    LABLAXIS: PKT_APID
+    VAR_TYPE: support_data
+
+seq_flgs:
+    <<: *default
+    CATDESC: CCSDS Packet Grouping Flags (3=not part of group)
+    FIELDNAM: Grouping Flags
+    LABLAXIS: SEQ_FLGS
+    VAR_TYPE: support_data
+
+src_seq_ctr:
+    <<: *default
+    CATDESC: CCSDS Packet Sequence Count (increments with each new packet)
+    FIELDNAM: Packet Sequence Count
+    LABLAXIS: SRC_SEQ_CTR
+    VAR_TYPE: support_data
+
+pkt_len:
+    <<: *default
+    CATDESC: CCSDS Packet Length (number of bytes after Packet length minus 1)
+    FIELDNAM: Packet Length
+    LABLAXIS: PKT_LEN
+    VAR_TYPE: support_data
+
+shcoarse:
+    <<: *default
+    CATDESC: Secondary Header - Whole-seconds part of SCLK
+    FIELDNAM: S/C Time - Seconds
+    LABLAXIS: SHCOARSE
+    VAR_TYPE: support_data
+
+packet_version:
+    <<: *default
+    CATDESC: Packet version - this will be incremented each time the format of the packet changes.
+    FIELDNAM: Packet Version
+    LABLAXIS: PACKET_VERSION
+    VAR_TYPE: support_data
+
+cmdexe:
+    <<: *default
+    lablaxis: CMDEXE
+    CATDESC: Number of commands that have been executed. Counts 0-255, then rolls over to 0.  Reset via CLR_LATCHED_SINGLE(COMMAND_COUNTS) [also resets cmdjrct, cmdacc, itf_error counts)
```

Comment:
You will need to move ones with long description to VAR_NOTES. Andriy pointed that out.

Reply:
Good catch! It looks like the character limit for `CATDESC` is 80 characters according to the ISTP website, so I will go by that.